io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl

```diff
@@ -0,0 +1,417 @@
+@prefix : <urn:samm:io.catenax.secondary_material_content:1.0.0#> .
+@prefix ext-part: <urn:samm:io.catenax.serial_part:1.0.1#> .
+
+:SecondaryMaterialContent a samm:Aspect ;
+   samm:preferredName "Secondary Material Content"@en ;
+   samm:description "This model refers to a part containing one or more different material. It references the share on primary and secondary material and relevant details."@en ;
+   samm:properties ( :secondaryMaterialContent :supplierID :orderNumber ext-part:manufacturerPartId ext-part:customerPartId ext-part:nameAtManufacturer ext-part:nameAtCustomer ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:secondaryMaterialContent a samm:Property ;
+   samm:preferredName "secondary Material Content"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
+   samm:characteristic :SecondaryMaterialContentMaterials .
+
+:supplierID a samm:Property ;
+   samm:preferredName "supplier ID"@en ;
+   samm:description "Suppliers individual business ID within the Catena-X network."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BPN1234567890ZZ" .
+
+:orderNumber a samm:Property ;
+   samm:preferredName "order Number"@en ;
+   samm:description "Order number as provided by supplier."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "856496" .
+
+:SecondaryMaterialContentMaterials a samm-c:Set ;
+   samm:preferredName "Secondary Material Content Materials"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
+   samm:dataType :SecondaryMaterialContentMaterialsEntity .
+
+:SecondaryMaterialContentMaterialsEntity a samm:Entity ;
+   samm:preferredName "Secondary Material Content Materials Entity"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
+   samm:properties ( :materialNameAtSupplier :materialClass :materialNameStandardized :unitOfMeasure :bioBased :inorganic :certificate ) .
+
+:materialNameAtSupplier a samm:Property ;
+   samm:preferredName "material name at supplier"@en ;
+   samm:description "Material name as provided by supplier."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Opatex" .
+
+:materialClass a samm:Property ;
+   samm:preferredName "material Class"@en ;
+   samm:description "Material class according to VDA standard 231-106."@en ;
+   samm:see <https://www.mdsystem.com/imdsnt/startpage/index.jsp> ;
+   samm:characteristic :MaterialClass ;
+   samm:exampleValue "5.1" .
+
+:materialNameStandardized a samm:Property ;
+   samm:preferredName "material name standardized"@en ;
+   samm:description "Material name according to standard."@en ;
+   samm:characteristic :MaterialNameStandardized .
+
+:unitOfMeasure a samm:Property ;
+   samm:preferredName "unit of measure"@en ;
```

Comment:
please check the remaining preferred names for uppercase writing and correct accordingly